test/functional/apps/management/_field_formatter.ts

```diff
@@ -478,36 +480,32 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       specs.forEach((spec, index) => {
         const fieldName = `${index}`;
-        it(
-          `edit field format of "${fieldName}" field to "${spec.applyFormatterType}"` +
-            spec.expectFormatterTypes
-            ? `, and check available formats types`
-            : '',
-          async () => {
-            await PageObjects.settings.filterField(fieldName);
-            await PageObjects.settings.openControlsByName(fieldName);
-            await PageObjects.settings.toggleRow('formatRow');
+        it(`edit field format of "${fieldName}" field to "${spec.applyFormatterType}"${
+          spec.expectFormatterTypes ? ', and check available formats types' : ''
```

Comment:
fixed name of the test. the expression was incorrect and some tests were named: `, and check available formats types` instead of a full name